### PR TITLE
Add version module and remove global reference

### PR DESCRIPTION
**Pull request by jwerle**
`Omnitone.VERSION` was left when https://github.com/GoogleChrome/omnitone/issues/14 should've removed it

### CODE REVIEW

**Comment by jwerle** [comment]
@hoch let me know if i need to rebase

**Comment by hoch** [comment]
@jwerle Joseph, is it common to have a separate file just for the version number?

**Comment by jwerle** [comment]
@hoch no, it isn't but there is a reference to the version. It seemed that the version may be of importance so i made a variable reference in a module to be used in multiple instances

**Comment by jwerle** [comment]
@hoch let me know what you think is best! This is currently breaking builds :/

File: src/foa-decoder.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ var FOAPhaseMatchedFilter = require('./foa-phase-matched-filter.js');
 var FOAVirtualSpeaker = require('./foa-virtual-speaker.js');
 var FOASpeakerData = require('./foa-speaker-data.js');
 var Utils = require('./utils.js');
+var VERSION = require('./version.js')
```
Comment (hoch) [inline src/foa-decoder.js]: Let's make this more explicit:

```
var SystemVersion = require('./version.js');
```

File: src/foa-decoder.js [hunk 2]
```diff
@@ -85,7 +86,7 @@ function FOADecoder (context, videoElement, options) {
  * @return {Promise}
  */
 FOADecoder.prototype.initialize = function () {
-  Utils.LOG('Version: ' + Omnitone.VERSION);
+  Utils.LOG('Version: ' + VERSION);
```
Comment (hoch) [inline src/foa-decoder.js]: Utils.LOG('Version: ' + SystemVersion);

File: src/omnitone.js [hunk 3]
```diff
@@ -32,12 +32,13 @@ var FOAPhaseMatchedFilter = require('./foa-phase-matched-filter.js');
 var FOAVirtualSpeaker = require('./foa-virtual-speaker.js');
 var FOADecoder = require('./foa-decoder.js');
 var Utils = require('./utils.js');
+var VERSION = require('./version.js')
```
Comment (hoch) [inline src/omnitone.js]: We don't need this here anymore.

File: src/omnitone.js [hunk 4]
```diff
@@ -32,12 +32,13 @@ var FOAPhaseMatchedFilter = require('./foa-phase-matched-filter.js');
...
 
 /**
```
Comment (hoch) [inline src/omnitone.js]: Line 37~41 can be removed.

**Comment by jwerle** [comment]
@hoch done! I've rebased/squashed this PR

File: src/foa-decoder.js [hunk 5]
```diff
@@ -27,6 +27,7 @@ var FOAPhaseMatchedFilter = require('./foa-phase-matched-filter.js');
 var FOAVirtualSpeaker = require('./foa-virtual-speaker.js');
 var FOASpeakerData = require('./foa-speaker-data.js');
 var Utils = require('./utils.js');
+var SystemVersion = require('./version.js')
```
Comment (hoch) [inline src/foa-decoder.js]: Missing semicolon.

File: build/omnitone.js [hunk 6]
```diff
@@ -800,6 +794,7 @@ return /******/ (function(modules) { // webpackBootstrap
 	var FOAVirtualSpeaker = __webpack_require__(7);
 	var FOASpeakerData = __webpack_require__(9);
 	var Utils = __webpack_require__(3);
+	var SystemVersion = __webpack_require__(10)
```
Comment (hoch) [inline build/omnitone.js]: I guess this is the artifact of a missing semicolon, Huh?

**Comment by jwerle** [comment]
@hoch ah yes oops! I just pushed an update

**Comment by hoch** [comment]
Thanks!

**Pull request merged**